exercises/series/src/Example.hs

```diff
@@ -4,5 +4,6 @@ import Data.Char (digitToInt)
 import Data.List (tails)
 
 slices :: Int -> String -> [[Int]]
+slices 0 _ = [[]]
```

Comment:
It we decide to test it, I would say to test it against `[[]]`. Otherwise, if the solution is not unique, I would prefer not to test it at all, because I cannot confidently say that those two are the only acceptable solutions.